docs/current_docs/features/security.mdx

```diff
@@ -0,0 +1,77 @@
+---
+slug: /features/security
```

Comment:
I have replaced the current Features/Secrets page with a revised Features/Security page that discusses the sandboxing feature + the original Secrets page content. A redirection is also in place for the old URL.

@jpadams @kpenfound please review